kasa/smartdevice.py

```diff
@@ -211,6 +211,7 @@ def __init__(
         #       checks in accessors. the @updated_required decorator does not ensure
         #       mypy that these are not accessed incorrectly.
         self._last_update: Any = None
+        self.discovery_info: Optional[Dict[str, Any]] = None
```

Comment:
```suggestion
        self._discovery_info: Optional[Dict[str, Any]] = None
```
Let's make it private.

Reply:
Ah! I couldn’t decide and in the end I got it wrong 😃. I’ll fix tomorrow.

Reply:
No worries :+1: My take is that it's usually better to "hide" internals and provide an API whenever it makes sense. This way the external API can remain the same even though some internals may change at some point.

Reply:
Done